Source/WebCore/accessibility/AccessibilityTableCell.cpp

```diff
@@ -322,7 +325,7 @@ AccessibilityObject* AccessibilityTableCell::titleUIElement() const
     // Table cells that are th cannot have title ui elements, since by definition
     // they are title ui elements
     Node* node = m_renderer->node();
-    if (node && node->hasTagName(thTag))
+    if (node && WebCore::elementName(node) == ElementName::HTML_th)
```

Comment:
Can be shortened to:

`if (WebCore::elementName(node) == ElementName::HTML_th)`